src/admin-views/settings/tickets-commerce/stripe/connect/active/webhooks.php

```diff
@@ -0,0 +1,56 @@
+<?php
```

Comment:
This looks like a copy-paste. We'll need to set a `@todo` to revisit the template and use the data we have for Stripe's connection.